File: GremsReliquary/src/main/java/gremsReliquary/relics/cursed/SternwardSword.java
package gremsReliquary.relics.cursed;

import com.badlogic.gdx.graphics.Texture;
import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction;
import com.megacrit.cardcrawl.core.AbstractCreature;
import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
import gremsReliquary.GremsReliquary;
import gremsReliquary.powers.CursedStrengthPower;
import gremsReliquary.util.TextureLoader;
import kotlinReliquary.relics.AbstractGremRelic;

public class SternwardSword extends AbstractGremRelic {
    public static final String ID = GremsReliquary.makeID(SternwardSword.class.getSimpleName());
    public static final Texture IMG = TextureLoader.getTexture("gremsReliquaryResources/images/relics/SternwardSword.png");
    public static final Texture OUTLINE = TextureLoader.getTexture("gremsReliquaryResources/images/relics/outline/SternwardSword.png");
    public static int amount = 2;
    
    AbstractCreature p = AbstractDungeon.player;
    
    public SternwardSword() {
        super(ID, IMG, OUTLINE, RelicTier.RARE, RelicType.CURSED, LandingSound.CLINK);
    }
    
    @Override
    public void onEquip() {
        
    }
    
    @Override
    public void atBattleStart() {
        AbstractDungeon.actionManager.addToTop(new RelicAboveCreatureAction(p, this));
        act(new ApplyPowerAction(p, p, new CursedStrengthPower(p, p, -3), -3));
    }
    
    // Description
    @Override
    public String getUpdatedDescription() {
        return DESCRIPTIONS[0];
    }
}

File: GremsReliquary/src/main/kotlin/kotlinReliquary/relics/cursed/SternwardSword.kt
package kotlinReliquary.relics.cursed

import com.megacrit.cardcrawl.actions.common.ApplyPowerAction
import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction
import com.megacrit.cardcrawl.dungeons.AbstractDungeon
import com.megacrit.cardcrawl.relics.AbstractRelic
import gremsReliquary.GremsReliquary
import gremsReliquary.powers.CursedStrengthPower
import gremsReliquary.util.TextureLoader
import kotlinReliquary.relics.AbstractGremRelic

class SternwardSword : AbstractGremRelic(ID, IMG, OUTLINE, AbstractRelic.RelicTier.RARE, AbstractGremRelic.RelicType.CURSED, AbstractRelic.LandingSound.CLINK) {
    companion object {
        val ID = GremsReliquary.makeID(SternwardSword::class.java.simpleName)
        val IMG = TextureLoader.getTexture("gremsReliquaryResources/images/relics/SternwardSword.png")
        val OUTLINE = TextureLoader.getTexture("gremsReliquaryResources/images/relics/outline/SternwardSword.png")
        var amount = 2
    }

    override fun onEquip() {

    }

    override fun atBattleStart() {
        AbstractDungeon.actionManager.addToTop(RelicAboveCreatureAction(p, this))
        act(ApplyPowerAction(p, p, CursedStrengthPower(p, p, -3), -3))
    }

    // Description
    override fun getUpdatedDescription(): String {
        return DESCRIPTIONS[0]
    }
}